src/raft.c

```diff
@@ -97,24 +100,211 @@ void entryAttachRaftReq(RedisRaftCtx *rr, raft_entry_t *entry, RaftReq *req)
 
 /* ----------------------------- Log Execution ------------------------------ */
 
+static bool isSharding(RedisRaftCtx *rr) {
```

Comment:
```suggestion
static bool isSharding(RedisRaftCtx *rr) 
{
```

Maybe, we don't even need a function?

Reply:
its simple, but could take more logic in future